builds.js

```diff
@@ -40,9 +40,8 @@ class Builds {
     this.warehouse = warehouse;
 
     options = Object.assign({}, defaultOptions, options);
-    if (options.cache && options.cache.enabled) {
-      this.cache = new Builds.Cache(this._get.bind(this), options.cache);
-    }
+    this.cache = new Cache({});
```

Comment:
This accepts options. We should pass them through from the client (e.g. `fsCachePath`).